sceptre/nlpsandbox/config/common/nlpsandboxvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/vpc-mini.yaml
+stack_name: nlpsandboxvpc
+stack_tags:
+  Department: "CompOnc"
+  Project: "NLP Sandbox"
+  OwnerEmail: "thomas.yu@sagebase.org"
+parameters:
+  VpcSubnetPrefix: "10.255.21"
+  VpcName: nlpsandboxvpc
+hooks:
+  before_launch:
+    - !cmd "wget https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/master/VPC/vpc-mini.yaml -N -P templates/remote"
```

Comment:
you may want to pin this to `v0.2.4`, just replace the s3 reference with 

```
https://github.com/Sage-Bionetworks/aws-infra/blob/v0.2.4/templates/VPC/vpc-mini.yaml
```

Reply:
I'll use the same [setup](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/blob/prod/config/prod/s3-cfn-templates.yaml#L9) as the Nextflow infra repo.